### PR TITLE
[UXD][AAP-25027] AWX - updates to Organizations section breadcrumbs & headers

### PR DESCRIPTION
**Pull request by tiyiprh**
https://issues.redhat.com/browse/AAP-25027

Updates to Organizations section breadcrumbs & headers section breadcrumbs & headers

### CODE REVIEW

File: cypress/e2e/awx/access/organizations.cy.ts [hunk 1]
```diff
@@ -86,18 +86,18 @@ describe('Organizations: Edit and Delete', function () {
     cy.get('[data-cy="name-column-cell"]').within(() => {
       cy.get('a').click();
     });
-    cy.verifyPageTitle(`${organization.name}`);
+    cy.verifyPageTitle(`Edit ${organization.name}`);
```
Comment (acruzgon) [inline cypress/e2e/awx/access/organizations.cy.ts:89]:
```suggestion
    cy.verifyPageTitle(organization.name);
```

File: cypress/e2e/awx/access/organizations.cy.ts [hunk 2]
```diff
@@ -86,18 +86,18 @@ describe('Organizations: Edit and Delete', function () {
...
     cy.containsBy('button', /^Edit organization/).click();
-    cy.verifyPageTitle('Edit Organization');
+    cy.verifyPageTitle(`Edit ${organization.name}`);
     cy.getByDataCy('name')
       .clear()
       .type('now-edited ' + `${stringRandom}`);
     cy.containsBy('button', /^Save organization/).click();
-    cy.verifyPageTitle('now-edited ' + `${stringRandom}`);
+    cy.verifyPageTitle(`Edit ${organization.name}`);
```
Comment (acruzgon) [inline cypress/e2e/awx/access/organizations.cy.ts:96]:
```suggestion
    cy.verifyPageTitle(organization.name);
```

File: cypress/e2e/awx/access/organizations.cy.ts [hunk 3]
```diff
@@ -86,18 +86,18 @@ describe('Organizations: Edit and Delete', function () {
...
     cy.getByDataCy('edit-organization').click();
     cy.getByDataCy('name').clear().type(`${organization.name}`);
     cy.containsBy('button', /^Save organization/).click();
-    cy.verifyPageTitle(`${organization.name}`);
+    cy.verifyPageTitle(`Edit ${organization.name}`);
```
Comment (acruzgon) [inline cypress/e2e/awx/access/organizations.cy.ts:100]:
```suggestion
    cy.verifyPageTitle(organization.name);
```

**Review by acruzgon — CHANGES_REQUESTED** [review]

**Review by MilanPospisil — DISMISSED** [review]
LGTM :+1

Ask please @acruzgon if the requested changes were correctly implemented.

**Review by acruzgon — DISMISSED** [review]
lgtm!

**Review by acruzgon — APPROVED** [review]
lgtm!

**Pull request merged**